packages/yoroi-extension/features/support/webdriver.js

```diff
@@ -81,7 +86,8 @@ function getFirefoxBuilder() {
      * The config is deprecated and may be removed in the future.
      */
     .setPreference('xpinstall.signatures.required', false)
-    .setPreference('extensions.webextensions.uuids', firefoxUuidMapping);
+    .setPreference('extensions.webextensions.uuids', firefoxUuidMapping)
+    .setPreference('devtools.console.stdout.content', true);
```

Comment:
If I'm not mistaken this puts all browser logs to the console and not to a log file. So maybe it is better to find a way put console logs into a log file and not clogging up the console?